packages/image_picker/image_picker_platform_interface/lib/src/types/media_selection_options.dart

```diff
@@ -0,0 +1,20 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import '../../image_picker_platform_interface.dart';
+
+/// Specifies options class for selecting items when using [ImagePickerPlatform.getMedia].
+class MediaSelectionOptions {
+  /// Construct a new MediaSelectionOptions instance.
+  MediaSelectionOptions({
+    this.imageOptions,
+    this.allowMultiple = true,
+  });
+
+  /// Options that will apply to images upon selection.
+  final ImageOptions? imageOptions;
+
+  /// Whether to allow for selecting multiple media. Defaults to true.
```

Comment:
I think it would be better to make this `required` so that we can't accidentally get the wrong behavior by forgetting to set it.